migration/migrations/course/20180709170000_verifyGraders2.py

```diff
@@ -0,0 +1,9 @@
+def up(config, conn, semester, course):
+    with conn.cursor() as cursor:
+        cursor.execute("ALTER TABLE ONLY gradeable_component_data DROP COLUMN IF EXISTS verifier_id")
+        cursor.execute("ALTER TABLE ONLY gradeable_component_data ADD COLUMN verifier_id character varying(255) NULL")
```

Comment:
keep the column names consistent with the rest of the columns in that table (i.e. `gcd_verifier_id`)